x-pack/plugins/apm/common/utils/apm_ml_anomaly_query.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function apmMlAnomalyQuery(detectorIndex: 0 | 1 | 2) {
```

Comment:
WDYT about renaming `x-pack/plugins/apm/common/utils/environment_query.ts` to `x-pack/plugins/apm/common/utils/queries.ts` and adding it there?

I am going to add `serviceNodeNameQuery` there as well.

Reply:
might be good to have that separate to prevent unnecessarily increasing the bundle size for the initial chunk

Reply:
Okay, in that case, can we keep bundle them in a folder then? Right now the utils folder is a bit of a mess with a lot of random files. Trying to create some order in the chaos :)